DB/academicmanager.sql

```diff
@@ -124,3 +125,23 @@ ALTER TABLE `course`
 --
 ALTER TABLE `person`
   MODIFY `personId` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=13;
+
+
+-- DROP the columns below as requested by G.Metaxas
+ALTER TABLE person DROP username,DROP password;
```

Comment:
Split this into two separate statements and add an if exists condition: eg ALTER TABLE Person DROP COLUMN IF EXISTS USERNAME: